docs/diga_api.md

```diff
@@ -0,0 +1,289 @@
+The information lists a field `Endpunkt_Kommunikationsstelle`.
+This is the base url for the endpoint for a specific insurer.
+There is no central DiGA endpoint for verifying and reimbursing DiGA apps and handling might differ across apis.
+Some apis might also not support billing as of now.
+The api response should include details on how to proceed with the billing request in this case, e.g. sending invoices by mail or post.
+
+Although insurance companies use their own apis, they do follow an openapi specification for the request which is listed as [DiGA-YAML-Datei (YAML)](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/digitale_gesundheitsanwendungen/technische_anlagen_aktuell_7/digaSP_1_0_05.yaml) at the same [gkv page](https://www.gkv-datenaustausch.de/leistungserbringer/digitale_gesundheitsanwendungen/digitale_gesundheitsanwendungen.jsp).
+This [documents](https://github.com/alex-therapeutics/diga-api-client/blob/main/ENDPOINT_STATUS.md) lists which apis are currently working with the client.
+
+# Request format
+
+According to the openapi specification the request contains 4 different parameters which are sent with `ContentType: multipart/form-data`:
+
+- `iksender` - Institutionskennzeichen (IK) of the DiGA manufacturer (an id required for payments)
+- `ikempfaenger` - IK of the insurer which can be found in the mapping file (`Kostentraegerkennung` - [section 5.2.2](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/digitale_gesundheitsanwendungen/technische_anlagen_aktuell_7/DiGA_Anlage_1_Technische_Anlage_zur_RL_V1.1_20210225.pdf))
+- `verfahren` - a code which discriminates requests for code verification and reimbursement
+- `nutzdaten` - additional information for the request, including the prescription code
+  - the `nutzdaten` are described later in more detail as this is the tricky bit of the request
+
+## Institutionskennzeichen - IK numbers
+
+Detailed information about the IK numbers can be found on the gkv page [Sonstige Leistungserbringer](https://www.gkv-datenaustausch.de/leistungserbringer/sonstige_leistungserbringer/sonstige_leistungserbringer.jsp) in the file [Gemeinsames Rundschreiben Institutionskennzeichen (PDF)](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/GR_IK_2020-06-01.pdf) (German only).
+
+> Das IK ist ein eindeutiges Merkmal für die Abrechnung medizinischer und rehabilitativer Leistungen mit den Trägern der Sozialversicherung [...]
+
+IK numbers are unique identifiers, 9 digits long, required for the billing of medical services and used as official identifiers.
+Therefore, every DiGA manufacturer must request an IK number.
+An IK number is also required when [requesting a certificate from ITSG](https://www.itsg.de/produkte/trust-center/zertifikat-beantragen/) which is required for encrypting and decrypting the `nutzdaten` of the request.
```

Comment:
That might not necessarily be true. I found that this can be also requested from some accounting tools where a `Betriebsnummer` could be enough. Will also add some more details on how long the request for IK number takes. Initial reply says 7-10 days.

Reply:
Sure! :)
I was only hoping that we can reduce the waiting time by requesting the IK number and the certificates from ITSG in parallel. Maybe for test requests it would have worked with a fake IK number (of the DiGA manufacturer)? So far we are blocked by the prerequisites which also makes writing the documentation harder.

Regarding the comment on `Betriebsnummer`, it seems like there are two cases for requesting the certificates:
- as `Sonstige Leistungserbringer` with IK number - that should be our case
- as `Arbeitgeber` with `Betriebsnummer`

So I doubt that we can speed this up...